src/Microsoft.NetCore.Analyzers/Core/Security/DoNotUseInsecureDeserializerBinaryFormatterWithoutBinder.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.NetCore.Analyzers.Security.Helpers;
+
+namespace Microsoft.NetCore.Analyzers.Security
+{
+    /// <summary>
+    /// For detecting deserialization with <see cref="System.Runtime.Serialization.Formatters.Binary.BinaryFormatter"/> when its Binder property is not set.
+    /// </summary>
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public class DoNotUseInsecureDeserializerBinaryFormatterWithoutBinder : DoNotUseInsecureDeserializerWithoutBinderBase
```

Comment:
>DoNotUseInsecureDeserializerBinaryFormatterWithoutBinder [](start = 17, length = 56)

Is this the only implementation of `DoNotUseInsecureDeserializerWithoutBinderBase` which uses flow analysis?

Reply:
So far, I'll have at least one more for NetDataContractSerializer, and there should be others.